environment.yml

```diff
@@ -12,12 +12,12 @@ channels:
     - litex-hub
 dependencies:
     - litex-hub::capnproto-java=0.1.5_0012_g44a8c1e=20201104_165332
-    - litex-hub::prjxray-tools=0.1_2842_g6867429c=20210301_104249
-    - litex-hub::prjxray-db=0.0_252_g8372b58=20210622_220029
-    - litex-hub::yosys=0.9_5457_gc6681508=20210615_141355_py38
-    - litex-hub::nextpnr-fpga_interchange=v0.0_3690_g2c459961=20210625_074838
-    - litex-hub::iverilog=s20150603_0957_gad862020=20201120_145821
-    - litex-hub::prjoxide=v0.0_376_gd0b0340=20210624_125028
+    - litex-hub::prjxray-db=0.0_252_g8372b58=20210730_085634
```

Comment:
Did you really have to update all those conda packages?

Reply:
Yosys had to be updated, reset was updated but it wasn't necessary for PR to work.
New packages don't break anything so I'm leaving new versions